### PR TITLE
Update the Romanian translations

### PR DESCRIPTION
**Pull request by alexfikl**
This adds a few more strings to the Romanian translation. They're somewhat inspired by the French translations, since that's a similar-ish language that I also know.

This is my first time translating, so I'm not quite sure what the best way to do it is (best tense to use, politeness levels, etc.). Let me know if anything looks particularly weird!

(And a big thank you for this project!)

### CODE REVIEW

**Comment by alexfikl** [comment]
I compiled the game and looked at a few translations and found an issue: it seems like the "î" symbol is just rendered as a "." when clicking Quit on the main screen.

**Comment by alexfikl** [comment]
Thank you very much for the quick response and the detailed explanations! It's very helpful.

> First I would suggest to read our part about specific PR size restrictions for translation PRs, which also has the explanation why we have this limit : [`master`/docs/TRANSLATION.md#sharing-your-translation-work](https://github.com/ihhub/fheroes2/blob/master/docs/TRANSLATION.md?rgh-link-date=2024-03-19T17%3A27%3A11Z#sharing-your-translation-work)

I'm more than happy to split this up into smaller and more digestible pieces. I missed that note in the docs because thought "I can skip this, I know how to make a PR!". Sorry about that! :disappointed: 
 
> To answer your question about specific Romanian symbols that are missing, this is because we are generating special symbols with code since they did not exist in the original font which was basically just ascii for the English version. We have added some but not all, and for buttons we have an entirely separate font that is completely generated on the fly by our engine. Adding the i with circumflex will be easy. This is for the normal text font. For button text font I don't think there is much coverage for Romanian symbols right now, but it is easy to add.

@zenseii  Thank you for the explanation! I'm happy to add these if it's not too hard and no special knowledge of the codebase is needed. Where would they go?

> I also noticed that you added some #fuzzy tags for strings that you have also changed the translation of. Basically what a fuzzy-tag means is that the use of the English string will be forced to be shown because the translation still needs work, and so really there is no reason for a translator to add or remove a fuzzy tag if the translation hasn't changed or isn't ready.

Yes, that was the intention. I mostly wasn't sure in what context those strings were used and what they meant, so the translation has a good chance of being wrong. Should I just leave the translation empty then?

> Another thing I noticed is that you have changed the translator which our standard in this project is to leave it as fheroes2 team because there are always different people who contribute to a translation so changing this back and forth makes no sense and avoids any discussions about this
>
> Lastly, there are two empty lines added in your PO file that don't need to be there:

Oops! That's what I get for using a UI and not checking the modifications too much! I'll revert these.

**Comment by alexfikl** [comment]
Thank you for the followup! I've reduced the PR to a more reasonable size without any of the mentioned issues.

> You would need to be familiar with C++ to do this and have some understanding of the codebase. However, it would mostly be a copy-paste job. The code for the letter generation for regular text would go in this function.

That's all fine. I can take a look! :grin:

File: files/lang/ro.po [hunk 1]
```diff
@@ -29,164 +29,183 @@ msgid ""
...
+"JOC\n"
+"STANDARD"
 
 msgid ""
 "CAMPAIGN\n"
 "GAME"
-msgstr ""
+msgstr "CAMPANIE"
 
+#, fuzzy
```
[the remark this answers is not in the document]
Comment (alexfikl) [reply files/lang/ro.po]: Yep! There was another one below :facepalm: 
https://github.com/ihhub/fheroes2/compare/4852c11ff478ebc93d08bd55a958fbcd04cbf767..688f61a5ac9af95e8198821532383f7dac84771f

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@alexfikl , thank you so much for this update!

**Pull request merged**